go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/kanisterio/kanister
 
 go 1.22
 
-toolchain go1.22.2
+toolchain go1.22.4
```

Comment:
This would address the most recent vulnerabilities flagged by `govulncheck`.

Does it does address the risk described [in this comment](https://github.com/kanisterio/kanister/pull/2931#issuecomment-2174701867)?